### PR TITLE
feat: update item language selector

### PR DESCRIPTION
**Pull request by LeoniePeters**
- Refactor ItemLanguageSelector template into one dropdown
  - Redirects to login before showing the dropdown (clicking the toggle button)
  - When language is selected it mentions the language in the toggle button and adds an option to the dropdown to remove the selected language
  - Mobile: no text and status indicator up to small breakpoint
  - When there is an error, the dropdown is disabled and there is a helptext underneath
- Moves the updated ItemLanguageSelector  component down the item page
- Refactor `swiper-height` SCSS mixin
  - Renamed to `media-viewer-height`
  - no longer needs to account for extra height when closing the language selector
  - adds the height available now the language selector is moved down to the `media-viewer-height` (`3.5rem`)
  - Extended with styles to prevent feedback button overlap
- Use `media-viewer-height` mixin for embeds
- Adds ItemLanguageSelector to the style guide

### CODE REVIEW

File: packages/portal/src/components/item/ItemLanguageSelector.vue [hunk 1]
```diff
@@ -109,20 +76,29 @@
...
         default: false
       },
+      /**
+       * Language the item is translated to
+       */
       translationLanguage: {
         type: String,
         default: null
+      },
+      behindLogin: {
```
Comment (lbiedinger) [inline packages/portal/src/components/item/ItemLanguageSelector.vue:93]: is this only used to allow the setup in the styleguide?

File: packages/portal/src/components/item/ItemLanguageSelector.vue [hunk 2]
```diff
@@ -1,101 +1,68 @@
 <template>
-  <transition
-    name="fade"
-  >
-    <b-container
-      fluid
-      class="d-flex border-bottom align-items-center"
+  <div>
+    <b-dropdown
+      no-flip
```
[the remark this answers is not in the document]
Comment (LeoniePeters) [reply packages/portal/src/components/item/ItemLanguageSelector.vue]: Indeed. I've made that so and adjusted the styles for when it opens to the top.

File: packages/portal/src/components/item/ItemLanguageSelector.vue [hunk 3]
```diff
@@ -1,101 +1,68 @@
...
       data-qa="item language selector"
+      :disabled="fromTranslationError"
+      @show="login"
```
[the remark this answers is not in the document]
Comment (LeoniePeters) [reply packages/portal/src/components/item/ItemLanguageSelector.vue]: Done.

File: packages/portal/src/components/item/ItemLanguageSelector.vue (hunk 1 above)
Comment (LeoniePeters) [reply packages/portal/src/components/item/ItemLanguageSelector.vue]: Yes. Now that we moved the login to the options this is no longer needed.

**Review by lbiedinger — APPROVED** [review]

**Pull request merged**